exchange/exchange.go

```diff
@@ -1187,3 +1199,9 @@ func buildStoredAuctionResponse(storedAuctionResponses map[string]json.RawMessag
 
 	return adapterBids, liveAdapters, nil
 }
+
+func callSignHeader(experiment *openrtb_ext.Experiment, info config.BidderInfo) bool {
```

Comment:
Consider adding a unit test for `callSignHeader` to exercise all of the emnedded logic statements.

Reply:
Added